### PR TITLE
Updating dependencies in README

### PR DESCRIPTION
**Pull request by idorrington-dstl**
adding orderedSet to dependencies

### CODE REVIEW

**Review by jswright-dstl — APPROVED** [review]

**Comment by sdhiscocks** [comment]
Worth adding all the ones that are missing, not just `ordered_set`.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Stone Soup uses the following dependencies:
 | [ruamel.yaml](https://yaml.readthedocs.io/) | MIT |
 | [pymap3d](https://github.com/scivision/pymap3d) | MIT |
 | [utm](https://github.com/Turbo87/utm) | MIT |
+| [OrderedSet](https://github.com/simonpercivall/orderedset) | PSFL |
```
Comment (sdhiscocks) [inline README.md:39]: This should be `ordered-set`. See `setup.py`

**Review by sdhiscocks — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -31,11 +31,15 @@ Stone Soup uses the following dependencies:
 | Name | License |
 | ---- | ------- |
 | [Python](https://www.python.org/) (v3.6+) | PSFL |
+| [numpy](https://numpy.org/) | BSD |
 | [SciPy](https://www.scipy.org/) | BSD |
 | [matplotlib](https://matplotlib.org/) | [PSF/BSD-compatible](https://matplotlib.org/users/license.html) |
 | [ruamel.yaml](https://yaml.readthedocs.io/) | MIT |
 | [pymap3d](https://github.com/scivision/pymap3d) | MIT |
 | [utm](https://github.com/Turbo87/utm) | MIT |
+| [ordered-set](https://github.com/simonpercivall/orderedset) | PSFL |
```
Comment (sdhiscocks) [inline README.md:40]: Link/license is wrong: https://github.com/LuminosoInsight/ordered-set

**Review by sdhiscocks — CHANGES_REQUESTED** [review]

**Review by sdhiscocks — APPROVED** [review]
Thanks. If you could just squash your commits that'd be great.

**Pull request merged**